### DISCUSSION TITLE
Legibilidade no Terminal - git diff

### DISCUSSION THREAD

**Discussion by marcoblos** [frontendbr/forum#402] [Discussão]
E ai galera, tudo certo?

Antes de mais nada, aviso que essa é uma discussão baseada em opiniões pessoais. hahahaha

Atualmente utilizo muito o terminal do Linux, sendo uma das principais finalidades utilizar o Git, mas tem uma coisa que sempre me deixou um pouco chateado, o diff default. Hoje olhando essa [lista de 50 tecnologias de devtools](https://stackshare.io/posts/top-developer-tools-2016?utm_content=stackshare_awards&utm_campaign=eoy_2016&utm_source=StackShare%20Marketing&utm_medium=email#devops) encontrei uma que acabou me agradando e resolveu o problema de "diff feio que não dá pra entender", a ferramenta se chama [git diff so fancy](https://github.com/so-fancy/diff-so-fancy).

Já que encontrei solução para resolver o meu problema de diff, vocês tem alguma sugestão ou alguma outra ferramenta que ajuda a melhorar a "legibilidade" no terminal?

**Comment by felipefialho** [comment]
c/c @matmarsiglio

**Comment by mtmr0x** [comment]
@marcoblos 

`sudo apt-get install tig`
seja feliz haha.

E se tu usa VIM, eis um cara que me ajuda MUITO: https://github.com/tpope/vim-fugitive.

E tem uma issue criada aqui há um tempo sobre logs fancy de GIT, que na verdade era sobre aliases, que o @willianjusten tem uns lindões, [esse aqui](https://github.com/frontendbr/forum/issues/373). 

Eu uso um lg que peguei de um amigo e incrementei uma coisinha só. Se tu digitar no seu terminal:

```
git config --global alias.lg "log --graph --pretty=format:'%Cred%h%Creset -%C(yellow)%d%Creset %s %Cgreen(%cr) %C(bold blue)<%an>%Creset' --abbrev-commit --date=relative"
```

Eu gosto do meu terminal super puro, sou um amante do Linux e amo o terminal do Ubuntu, no meu dotfiles tem umas coisinhas simples, mas que eu gosto bastante, dá uma olhada lá nos meus dotfiles, talvez veja algo do seu interesse:

[https://github.com/matmarsiglio/dotfiles](https://github.com/matmarsiglio/dotfiles)

1 upa.
